### DISCUSSION TITLE
LRU Cache

### DISCUSSION THREAD

**Discussion by ajityadav2621** [community/community#145461] [Programming Help]
### Select Topic Area

General

### Body

Anyone here working on Designing LRU Cache based engine, I was working on feature to optimize the Database memory using thread-safe binary search tree (BST). I would love to connect with fellows, who has expertise in working on complex problem statement. 

Thank you! Let's connect.

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @AlexAjit , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by ajityadav2621** [reply]
@shinybrightstar Sure, got your point. I will be giving more details once I am done with some of my research, I had some doubts but got cleared and thank you for giving me suggestions, highly appreciated.
